src/routes/(site)/videos/+page.server.ts

```diff
@@ -1,29 +1,42 @@
+import playListOrderBy from '$lib/videos/playlistOrderBy';
+
 export const load = async function ({ locals }) {
-	const playlists = await locals.prisma.playlist.findMany({
+	const playlistMeta = await locals.prisma.playlist.findMany({
```

Comment:
This is one of the questionable queries. Have to get all playlists _first_, then get each playlist in a separate query.